docs/file-reference/index.md

```diff
@@ -75,6 +77,25 @@ metadata:
 |----------------|------------------------------------|---------------------|
 | metadataObject | [metadataObject](#metadata-object) | Metadata to be used |
 
+## starter projects
+
+> Example using a [Quarkus quickstart project](https://github.com/quarkusio/quarkus-quickstarts)
+
+```yaml
+starterProjects:
+  - name: quarkus-quickstart
+    git:
+      location: https://github.com/quarkusio/quarkus-quickstarts
```

Comment:
you're missing adding the `description` key to this example

Reply:
added